include/aspect/simulator.h

```diff
@@ -1093,13 +1084,36 @@ namespace aspect
 
       /**
        * Remove the angular momentum of the given vector
+       *  
+       * @param relevant_dst locally relevant vector for the whole FE, will be
+       * filled at the end.
+       * @param tmp_distributed_stokes only contains velocity and pressure.
+       * @param use_constant_density determines whether to use a constant density
+       * (which corresponds to removing a net rotation instead of net angular
+       * momentum). 
+       *
+       * This function is implemented in
+       * <code>source/simulator/nullspace.cc</code>.
        */
-      void remove_net_angular_momentum( LinearAlgebra::BlockVector &relevant_dst, LinearAlgebra::BlockVector &tmp_distributed_stokes);
+      void remove_net_angular_momentum( LinearAlgebra::BlockVector &relevant_dst, 
+                                        LinearAlgebra::BlockVector &tmp_distributed_stokes,
+                                        bool use_constant_density);
```

Comment:
Can you reorder this so that input arguments come before output arguments (and mark input args as `const`)?